tests/integration/tests/istio_config_test.go

```diff
@@ -96,10 +96,20 @@ func TestIstioConfigDetails(t *testing.T) {
 	}
 }
 
+func TestIstioConfigInvalidName(t *testing.T) {
+	name := "invalid"
+	assert := assert.New(t)
+	config, code, _ := utils.IstioConfigDetails(utils.BOOKINFO, name, kubernetes.VirtualServices)
+	assert.NotEqual(200, code)
+	assert.Empty(config)
+}
+
 func TestIstioConfigPermissions(t *testing.T) {
 	assert := assert.New(t)
 	perms, err := utils.IstioConfigPermissions(utils.BOOKINFO)
 
 	assert.Nil(err)
 	assert.NotEmpty(perms)
+	assert.NotEmpty((*perms)[utils.BOOKINFO])
+	assert.NotEmpty((*(*perms)[utils.BOOKINFO])["authorizationpolicies"])
```

Comment:
This is an odd syntax that I've never seen before. I think it stems from the models themselves though and the fact that they are pointers to maps so nothing to change here.